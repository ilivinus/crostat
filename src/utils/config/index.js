export const API_KEY = { 
    key_name : 'X-CMC_PRO_API_KEY', 
    value : '57440339-3174-489c-8cc3-54ed63a243df' 
}