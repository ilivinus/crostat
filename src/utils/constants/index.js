import { Platform } from 'react-native'

export const PAGE_SIZE = 20;
export const DEBOUNCE_INTERVAL = 2000;
export const colors = {
    red: '#F8333C',
    green: '#4ADF86',
    darkGray: '#31393C',
    greyOutline: '#bbbbbb',
    primary: '#00C0FF',
    primaryLight: 'rgba(0, 192, 255, 0.15)',
    lightGrey: '#B4B4B4',
    white: '#ffffff',
    transparent: 'transparent',
    darkBlue: '#161C36',
    black: '#000',
    selectionColor: Platform.OS === 'ios' ? '#00C0FF' : undefined
  };
  
  /**
   * App themes
   */
  export const themes = {
    dark: {
      ...colors,
      tabBarColor: '#192543',
      cardBackground: colors.darkBlue,
      headerTitleColor: colors.white,
      textColor: colors.white,
      thumbTintColor: colors.white,
    },
    light: {
      ...colors,
      tabBarColor: '#F3F3F3',
      cardBackground: colors.white,
      headerTitleColor: colors.darkGray,
      textColor: colors.darkGray,
      thumbTintColor: colors.darkGray,
    },
  };
  export const metrics = {
    //hitSlop: makeHitSlop,
    shadow: {
      basic: {
        shadowOffset: {
          width: 2,
          height: 0
        },
        shadowOpacity: 0.2,
        shadowColor: colors.black,
        shadowRadius: 6
      }
    }
  };
 export const properties = {
    feedbackEmail: {
      email: 'umeadumalivinus@gmail.com',
      subject: 'CloseCross App feedback',
    },
  };
  