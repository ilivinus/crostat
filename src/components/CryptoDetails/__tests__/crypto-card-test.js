import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import renderer from 'react-test-renderer';
import CryptoCard from '../presenter';
import CryptoCardComponent from '../index';
import { themes } from '../../../utils/constants'

Enzyme.configure({ adapter: new Adapter() })

describe('#CryptoDetails', () => {

    it('should render the presenter properly', () => {
        const component = renderer.create(<CryptoCard />).toJSON();

        expect(component).toMatchSnapshot();
    })
    it('should render the the index', () => {
        const component = renderer.create(<CryptoCard />).toJSON();

        expect(component).toMatchSnapshot();
    })
})

describe('#CryptoCardComponent', () => {
    let props;

    beforeEach(() => {

        const mockPercent1hColor = jest.fn();
        const mockPercent24hColor = jest.fn();
        const mockPercent7dColor = jest.fn();
        props = {
            screenProps: {
                theme: themes.dark,
            },
            crypto: { priceUsd: 232 },
            _percent1hColor: mockPercent1hColor,
            _percent24hColor: mockPercent24hColor,
            _percent7dColor: mockPercent7dColor,
            
        }


    })

    it('should render cryptocardComponent', () => {
        
        const mockOnRefresh = jest.fn();
        const component = renderer.create(<CryptoCardComponent _onRefresh={mockOnRefresh} {...props} />).toJSON();

        expect(component).toMatchSnapshot();
    })

    it('should check if onRefresh is called', () => {
        //arrange
        const mockOnRefresh = jest.fn();        
        const cryptoComponent = shallow(<CryptoCardComponent _onRefresh={mockOnRefresh} {...props} />);
        const instance = cryptoComponent.instance();
        //act
        instance._onRefresh();
        //assert
        expect(mockOnRefresh).toHaveBeenCalled();
        expect(mockOnRefresh).toHaveBeenCalledTimes(1);

    })

});
