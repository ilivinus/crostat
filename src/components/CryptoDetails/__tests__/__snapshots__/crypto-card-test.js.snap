// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#CryptoCardComponent should render cryptocardComponent 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
      },
      Object {
        "backgroundColor": "#161C36",
      },
    ]
  }
>
  <RCTScrollView
    contentContainerStyle={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "column",
        "paddingVertical": 1,
      }
    }
    refreshControl={
      <RefreshControlMock
        onRefresh={[Function]}
        refreshing={false}
        tintColor="#00C0FF"
      />
    }
  >
    <RCTRefreshControl />
    <View>
      <View
        style={
          Object {
            "height": 50,
            "marginVertical": 1,
            "width": "95%",
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignSelf": "stretch",
                "borderRadius": 0,
                "flex": 1,
                "flexDirection": "row",
                "justifyContent": "space-between",
                "padding": 5,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 0,
                  "width": 2,
                },
                "shadowOpacity": 0.2,
                "shadowRadius": 6,
              },
              Object {
                "backgroundColor": "#192543",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-start",
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 14,
                    "fontWeight": "500",
                  },
                  Object {
                    "color": "#B4B4B4",
                  },
                ]
              }
            >
              Name
            </Text>
          </View>
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-end",
              }
            }
          >
            <Text
              numberOfLines={1}
              style={
                Array [
                  Object {
                    "fontSize": 15,
                    "fontWeight": "400",
                  },
                  Object {
                    "color": "#ffffff",
                  },
                ]
              }
            />
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "height": 50,
            "marginVertical": 1,
            "width": "95%",
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignSelf": "stretch",
                "borderRadius": 0,
                "flex": 1,
                "flexDirection": "row",
                "justifyContent": "space-between",
                "padding": 5,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 0,
                  "width": 2,
                },
                "shadowOpacity": 0.2,
                "shadowRadius": 6,
              },
              Object {
                "backgroundColor": "#192543",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-start",
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 14,
                    "fontWeight": "500",
                  },
                  Object {
                    "color": "#B4B4B4",
                  },
                ]
              }
            >
              Rank
            </Text>
          </View>
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-end",
              }
            }
          >
            <Text
              numberOfLines={1}
              style={
                Array [
                  Object {
                    "fontSize": 15,
                    "fontWeight": "400",
                  },
                  Object {
                    "color": "#ffffff",
                  },
                ]
              }
            />
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "height": 50,
            "marginVertical": 1,
            "width": "95%",
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignSelf": "stretch",
                "borderRadius": 0,
                "flex": 1,
                "flexDirection": "row",
                "justifyContent": "space-between",
                "padding": 5,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 0,
                  "width": 2,
                },
                "shadowOpacity": 0.2,
                "shadowRadius": 6,
              },
              Object {
                "backgroundColor": "#192543",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-start",
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 14,
                    "fontWeight": "500",
                  },
                  Object {
                    "color": "#B4B4B4",
                  },
                ]
              }
            >
              Market Cap
            </Text>
          </View>
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-end",
              }
            }
          >
            <Text
              numberOfLines={1}
              style={
                Array [
                  Object {
                    "fontSize": 15,
                    "fontWeight": "400",
                  },
                  Object {
                    "color": "#ffffff",
                  },
                ]
              }
            >
              0
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "height": 50,
            "marginVertical": 1,
            "width": "95%",
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignSelf": "stretch",
                "borderRadius": 0,
                "flex": 1,
                "flexDirection": "row",
                "justifyContent": "space-between",
                "padding": 5,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 0,
                  "width": 2,
                },
                "shadowOpacity": 0.2,
                "shadowRadius": 6,
              },
              Object {
                "backgroundColor": "#192543",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-start",
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 14,
                    "fontWeight": "500",
                  },
                  Object {
                    "color": "#B4B4B4",
                  },
                ]
              }
            >
              Circulation Supply
            </Text>
          </View>
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-end",
              }
            }
          >
            <Text
              numberOfLines={1}
              style={
                Array [
                  Object {
                    "fontSize": 15,
                    "fontWeight": "400",
                  },
                  Object {
                    "color": "#ffffff",
                  },
                ]
              }
            />
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "height": 50,
            "marginVertical": 1,
            "width": "95%",
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignSelf": "stretch",
                "borderRadius": 0,
                "flex": 1,
                "flexDirection": "row",
                "justifyContent": "space-between",
                "padding": 5,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 0,
                  "width": 2,
                },
                "shadowOpacity": 0.2,
                "shadowRadius": 6,
              },
              Object {
                "backgroundColor": "#192543",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-start",
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 14,
                    "fontWeight": "500",
                  },
                  Object {
                    "color": "#B4B4B4",
                  },
                ]
              }
            >
              Max Supply
            </Text>
          </View>
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-end",
              }
            }
          >
            <Text
              numberOfLines={1}
              style={
                Array [
                  Object {
                    "fontSize": 15,
                    "fontWeight": "400",
                  },
                  Object {
                    "color": "#ffffff",
                  },
                ]
              }
            />
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "height": 50,
            "marginVertical": 1,
            "width": "95%",
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignSelf": "stretch",
                "borderRadius": 0,
                "flex": 1,
                "flexDirection": "row",
                "justifyContent": "space-between",
                "padding": 5,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 0,
                  "width": 2,
                },
                "shadowOpacity": 0.2,
                "shadowRadius": 6,
              },
              Object {
                "backgroundColor": "#192543",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-start",
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 14,
                    "fontWeight": "500",
                  },
                  Object {
                    "color": "#B4B4B4",
                  },
                ]
              }
            >
              Total Supply
            </Text>
          </View>
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-end",
              }
            }
          >
            <Text
              numberOfLines={1}
              style={
                Array [
                  Object {
                    "fontSize": 15,
                    "fontWeight": "400",
                  },
                  Object {
                    "color": "#ffffff",
                  },
                ]
              }
            >
              0
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "height": 50,
            "marginVertical": 1,
            "width": "95%",
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignSelf": "stretch",
                "borderRadius": 0,
                "flex": 1,
                "flexDirection": "row",
                "justifyContent": "space-between",
                "padding": 5,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 0,
                  "width": 2,
                },
                "shadowOpacity": 0.2,
                "shadowRadius": 6,
              },
              Object {
                "backgroundColor": "#192543",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-start",
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 14,
                    "fontWeight": "500",
                  },
                  Object {
                    "color": "#B4B4B4",
                  },
                ]
              }
            >
              Date added
            </Text>
          </View>
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-end",
              }
            }
          >
            <Text
              numberOfLines={1}
              style={
                Array [
                  Object {
                    "fontSize": 15,
                    "fontWeight": "400",
                  },
                  Object {
                    "color": "#ffffff",
                  },
                ]
              }
            />
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "height": 50,
            "marginVertical": 1,
            "width": "95%",
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignSelf": "stretch",
                "borderRadius": 0,
                "flex": 1,
                "flexDirection": "row",
                "justifyContent": "space-between",
                "padding": 5,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 0,
                  "width": 2,
                },
                "shadowOpacity": 0.2,
                "shadowRadius": 6,
              },
              Object {
                "backgroundColor": "#192543",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-start",
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 14,
                    "fontWeight": "500",
                  },
                  Object {
                    "color": "#B4B4B4",
                  },
                ]
              }
            >
              Price undefined
            </Text>
          </View>
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-end",
              }
            }
          >
            <Text
              numberOfLines={1}
              style={
                Array [
                  Object {
                    "fontSize": 15,
                    "fontWeight": "400",
                  },
                  Object {
                    "color": "#ffffff",
                  },
                ]
              }
            >
              232.00000000
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "height": 50,
            "marginVertical": 1,
            "width": "95%",
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignSelf": "stretch",
                "borderRadius": 0,
                "flex": 1,
                "flexDirection": "row",
                "justifyContent": "space-between",
                "padding": 5,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 0,
                  "width": 2,
                },
                "shadowOpacity": 0.2,
                "shadowRadius": 6,
              },
              Object {
                "backgroundColor": "#192543",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-start",
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 14,
                    "fontWeight": "500",
                  },
                  Object {
                    "color": "#B4B4B4",
                  },
                ]
              }
            >
              Last updated
            </Text>
          </View>
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-end",
              }
            }
          >
            <Text
              numberOfLines={1}
              style={
                Array [
                  Object {
                    "fontSize": 15,
                    "fontWeight": "400",
                  },
                  Object {
                    "color": "#ffffff",
                  },
                ]
              }
            />
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "height": 50,
            "marginVertical": 1,
            "width": "95%",
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignSelf": "stretch",
                "borderRadius": 0,
                "flex": 1,
                "flexDirection": "row",
                "justifyContent": "space-between",
                "padding": 5,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 0,
                  "width": 2,
                },
                "shadowOpacity": 0.2,
                "shadowRadius": 6,
              },
              Object {
                "backgroundColor": "#192543",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-start",
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 14,
                    "fontWeight": "500",
                  },
                  Object {
                    "color": "#B4B4B4",
                  },
                ]
              }
            >
              24h VOLUME
            </Text>
          </View>
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-end",
              }
            }
          >
            <Text
              numberOfLines={1}
              style={
                Array [
                  Object {
                    "fontSize": 15,
                    "fontWeight": "400",
                  },
                  Object {
                    "color": "#ffffff",
                  },
                ]
              }
            >
              0
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "height": 50,
            "marginVertical": 1,
            "width": "95%",
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignSelf": "stretch",
                "borderRadius": 0,
                "flex": 1,
                "flexDirection": "row",
                "justifyContent": "space-between",
                "padding": 5,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 0,
                  "width": 2,
                },
                "shadowOpacity": 0.2,
                "shadowRadius": 6,
              },
              Object {
                "backgroundColor": "#192543",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-start",
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 14,
                    "fontWeight": "500",
                  },
                  Object {
                    "color": "#B4B4B4",
                  },
                ]
              }
            >
              % CHANGE 1H
            </Text>
          </View>
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-end",
              }
            }
          >
            <Text
              numberOfLines={1}
              style={
                Array [
                  Object {
                    "fontSize": 15,
                    "fontWeight": "400",
                  },
                  Object {
                    "color": undefined,
                  },
                ]
              }
            >
              0
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "height": 50,
            "marginVertical": 1,
            "width": "95%",
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignSelf": "stretch",
                "borderRadius": 0,
                "flex": 1,
                "flexDirection": "row",
                "justifyContent": "space-between",
                "padding": 5,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 0,
                  "width": 2,
                },
                "shadowOpacity": 0.2,
                "shadowRadius": 6,
              },
              Object {
                "backgroundColor": "#192543",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-start",
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 14,
                    "fontWeight": "500",
                  },
                  Object {
                    "color": "#B4B4B4",
                  },
                ]
              }
            >
              % CHANGE 24H
            </Text>
          </View>
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-end",
              }
            }
          >
            <Text
              numberOfLines={1}
              style={
                Array [
                  Object {
                    "fontSize": 15,
                    "fontWeight": "400",
                  },
                  Object {
                    "color": undefined,
                  },
                ]
              }
            >
              0
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "height": 50,
            "marginVertical": 1,
            "width": "95%",
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignSelf": "stretch",
                "borderRadius": 0,
                "flex": 1,
                "flexDirection": "row",
                "justifyContent": "space-between",
                "padding": 5,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 0,
                  "width": 2,
                },
                "shadowOpacity": 0.2,
                "shadowRadius": 6,
              },
              Object {
                "backgroundColor": "#192543",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-start",
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 14,
                    "fontWeight": "500",
                  },
                  Object {
                    "color": "#B4B4B4",
                  },
                ]
              }
            >
              % CHANGE 7D
            </Text>
          </View>
          <View
            style={
              Object {
                "alignSelf": "center",
                "justifyContent": "flex-end",
              }
            }
          >
            <Text
              numberOfLines={1}
              style={
                Array [
                  Object {
                    "fontSize": 15,
                    "fontWeight": "400",
                  },
                  Object {
                    "color": undefined,
                  },
                ]
              }
            >
              0
            </Text>
          </View>
        </View>
      </View>
    </View>
  </RCTScrollView>
</View>
`;

exports[`#CryptoDetails should render the presenter properly 1`] = `
<View
  style={
    Object {
      "height": 50,
      "marginVertical": 1,
      "width": "95%",
    }
  }
>
  <View
    style={
      Array [
        Object {
          "alignSelf": "stretch",
          "borderRadius": 0,
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "space-between",
          "padding": 5,
          "shadowColor": "#000",
          "shadowOffset": Object {
            "height": 0,
            "width": 2,
          },
          "shadowOpacity": 0.2,
          "shadowRadius": 6,
        },
        Object {
          "backgroundColor": undefined,
        },
      ]
    }
  >
    <View
      style={
        Object {
          "alignSelf": "center",
          "justifyContent": "flex-start",
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontSize": 14,
              "fontWeight": "500",
            },
            Object {
              "color": "#B4B4B4",
            },
          ]
        }
      />
    </View>
    <View
      style={
        Object {
          "alignSelf": "center",
          "justifyContent": "flex-end",
        }
      }
    >
      <Text
        numberOfLines={1}
        style={
          Array [
            Object {
              "fontSize": 15,
              "fontWeight": "400",
            },
            Object {
              "color": undefined,
            },
          ]
        }
      />
    </View>
  </View>
</View>
`;

exports[`#CryptoDetails should render the the index 1`] = `
<View
  style={
    Object {
      "height": 50,
      "marginVertical": 1,
      "width": "95%",
    }
  }
>
  <View
    style={
      Array [
        Object {
          "alignSelf": "stretch",
          "borderRadius": 0,
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "space-between",
          "padding": 5,
          "shadowColor": "#000",
          "shadowOffset": Object {
            "height": 0,
            "width": 2,
          },
          "shadowOpacity": 0.2,
          "shadowRadius": 6,
        },
        Object {
          "backgroundColor": undefined,
        },
      ]
    }
  >
    <View
      style={
        Object {
          "alignSelf": "center",
          "justifyContent": "flex-start",
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontSize": 14,
              "fontWeight": "500",
            },
            Object {
              "color": "#B4B4B4",
            },
          ]
        }
      />
    </View>
    <View
      style={
        Object {
          "alignSelf": "center",
          "justifyContent": "flex-end",
        }
      }
    >
      <Text
        numberOfLines={1}
        style={
          Array [
            Object {
              "fontSize": 15,
              "fontWeight": "400",
            },
            Object {
              "color": undefined,
            },
          ]
        }
      />
    </View>
  </View>
</View>
`;
